testsuites/store/src/main/java/org/eclipse/rdf4j/sail/RDFStoreTest.java

```diff
@@ -718,6 +726,7 @@ public void testQueryBindings()
 	}
 
 	@Test
+	@Ignore("test is fundamentelly flawed as it uses multithreaded access on the same connection")
```

Comment:
The thing is: we have other test suites that specifically focus on this. Both `SailConcurrencyTest` and `SailIsolationLevelTest` do extensive tests using multiple concurrent connections. So I don't think this particular test adds much value.